src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -968,6 +969,17 @@ Parameter Plugin::GetMetric(const std::string& name, const std::map<std::string,
     } else if (name == ov::caching_properties) {
         std::vector<ov::PropertyName> cachingProperties;
         return decltype(ov::caching_properties)::value_type(cachingProperties);
+    } else if (name == ov::device::architecture) {
```

Comment:
https://github.com/openvinotoolkit/openvino/pull/13801#discussion_r1017920791
I added `ov::device::architecture` in supported properties. Thank you.